### PR TITLE
galaxy-objectstore: initial commit

### PR DESCRIPTION
**Pull request by bernt-matthias**
TODO/needed

- [x] new pypi release including `test-requirements.txt`

Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by natefoo** [comment]
galaxy-objectstore 20.9.1 has been released and should fix the error.

File: recipes/galaxy-objectstore/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% set version = "20.9.1" %}
+{% set sha256 = "3cdc27ddcdf891142a6af0601cf0490fc3c0ecc28e343c140500b7da8403412b" %}
+{% set galaxy_version = version.split(".")[:2]|join(".") %}
+
+package:
+  name: galaxy-objectstore
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/09/5b/26662352bddaf874997fae4014218548bd4eb847eb86f9a0f9a61ffd9f4b/galaxy-objectstore-{{ version }}.tar.gz
```
Comment (natefoo) [inline recipes/galaxy-objectstore/meta.yaml:10]:
```suggestion
  url: https://files.pythonhosted.org/packages/05/89/b1964bda19f06f497b2179f98a56810a5bf66fc4e72dbfee79a053022fa4/galaxy-objectstore-{{ version }}.tar.gz
  
  
```

Comment (bgruening) [reply recipes/galaxy-objectstore/meta.yaml]: Those URLs are actually better: https://pypi.io/packages/source/g/galaxy-objectstore/galaxy-objectstore-20.9.1.tar.gz

We can use than the bot to update the recipes :)

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Pull request merged**